contracts/src/DaypassPaymaster.sol

```diff
@@ -59,19 +54,19 @@ contract DaypassPaymaster is BasePaymaster {
         uint256 tokenCount = daypassContract.balanceOf(userOp.sender);
 
         if (tokenCount < 1) {
-            revert NoDaypass();
+            revert("no daypass");
```

Comment:
Switching to revert messages otherwise the entrypoint contract doesn't show the message :(